src/main/java/io/wispforest/owo/ui/hud/Hud.java

```diff
@@ -40,6 +40,7 @@ public class Hud {
      */
     public static void add(Identifier id, Supplier<Component> component) {
         pendingActions.add(flowLayout -> {
+            if (hasComponent(id)) return;
```

Comment:
honestly, `Hud#add`s for already existing components being no-ops seems kinda weird.
imo if you add a new HUD for an existing id it should either:
1. fully replace the old HUD, or
2. add the id into a list of active ids immediately and throw an exception if a duplicate HUD component is added